packages/example/test/example_spec.js

```diff
@@ -1,15 +1,31 @@
 let example = require('../index')
 let expect = require('chai').expect
-let normalize = require('path').normalize
+const { join, normalize, sep } = require('path')
+const { chain, curry } = require('lodash')
```

Comment:
we typically assign `_` and call methods off that since it's more clear the fn comes from lodash

Reply:
same with `path`

Reply:
I wondered what would be the approach, but found it used in both ways in the codebase...somewhere. So the team's preferred method is to import the default and not deconstruct the items? If that is the case, I can change these.

Reply:
yes, we typically use the default import